src/components/musette/musette.js

```diff
@@ -0,0 +1,143 @@
+export default function lesMusettes() {
+	const lesMusettes = document.querySelectorAll(".js-la-musette");
+
+	lesMusettes.forEach((laMusette) => {
+		const preventClick = (e) => {
+			e.preventDefault();
+			e.stopImmediatePropagation();
+		};
+		let mouseIsDown = false;
+		var mouseIsBeingDragged = false;
+		let startPositionX;
+		let scrollLeft;
+
+		// create wrapper
+		var musetteWrapper = document.createElement("div");
+		musetteWrapper.classList.add("musette-wrapper");
+
+		// create buttons
+		var leftButton = document.createElement("button");
+		leftButton.classList.add("musette-move-left");
+		var rightButton = document.createElement("button");
+		rightButton.classList.add("musette-move-right");
+
+		// insert wrapper before musette
+		laMusette.parentNode.insertBefore(musetteWrapper, laMusette);
+
+		// put elements into wrapper
+		musetteWrapper.appendChild(laMusette);
+		musetteWrapper.appendChild(leftButton);
+		musetteWrapper.appendChild(rightButton);
+
+		// get dimensions, used exclusively for buttons
+		// imported from old Madaveousel and maybe more specs than we need
+		const getMusetteSpecs = (laMusette) => {
```

Comment:
i think @JosephCurley wanted to use https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API ?